src/block-management/block-holder.js

```diff
@@ -19,6 +19,7 @@ type PropsType = BlockType & {
 	onChange: ( clientId: string, attributes: mixed ) => void,
 	onToolbarButtonPressed: ( button: number, clientId: string ) => void,
 	onBlockHolderPressed: ( clientId: string ) => void,
+	insertBlocksAfter: ( clientId: string, blocks: Array<Object> ) => void,
```

Comment:
I think we need to remove the `clientId` parameter from the definition here. Essentially, what we will be passing down to the block-holder is a function with the `clientId` string already "embedded" in it so, we will just call it by passing the new blocks.